complex/ft_config.c

```diff
@@ -669,7 +669,7 @@ void fts_cur_info(struct ft_series *series, struct ft_info *info)
 	info->class_function = set->class_function[series->cur_func];
 	info->op = set->op[series->cur_op];
 	info->test_flags = set->test_flags;
-	info->caps = set->caps[series->cur_caps];
+	info->caps = set->caps[series->cur_caps] | FT_CAP_MSG;
```

Comment:
I think for now it would be best to just use MSG capabilities (especially since it seems like psm2 fails if this is left out) and figure out a way to test the capabilities independently later, likely through the config file.